playbooks/support/on-call-slo.md

```diff
@@ -0,0 +1,31 @@
+# On-Call SLO
+
+This document describes a
```

Comment:
@dblandin updated this doc to be more explicit about our SLO (as referring to our response time). Is it clearer?

Reply:
Yes, that looks great!! :tada: